features/data_converter/binary/feature.java

```diff
@@ -1,34 +1,30 @@
 package data_converter.binary;
 
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.google.protobuf.util.JsonFormat;
 import io.temporal.api.common.v1.Payload;
 import io.temporal.sdkfeatures.Feature;
 import io.temporal.sdkfeatures.Run;
 import io.temporal.sdkfeatures.Runner;
 import io.temporal.workflow.WorkflowInterface;
 import io.temporal.workflow.WorkflowMethod;
-
-import static org.junit.jupiter.api.Assertions.assertArrayEquals;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
 import java.nio.file.Files;
 import java.nio.file.Paths;
 
-import com.google.protobuf.util.JsonFormat;
-
 @WorkflowInterface
 public interface feature extends Feature {
-    static byte[] deadbeef = new byte[]{(byte)0xde, (byte)0xad, (byte)0xbe, (byte)0xef};
-    
+  static byte[] deadbeef = new byte[] {(byte) 0xde, (byte) 0xad, (byte) 0xbe, (byte) 0xef};
```

Comment:
nit: `static` is redundant here, also this variable is a constant so convention is all caps with underscore to separate words

Reply:
I know you didn't write this code, but you did the same thing in some of the other features. Not a blocker

Reply:
Thanks, it makes sense, done...